src/assets/scss/pages/_not-found.scss

```diff
@@ -0,0 +1,31 @@
+.not-found {
+  position: absolute;
+  margin: auto;
+  top: 0;
+  right: 0;
+  bottom: 10;
+  left: 0;
+  width: 37.5rem;
+  height: 18.75rem;
+
+  &__panel {
+    background: #eee;
+    padding: 2.5rem 2.5rem;
+    border-radius: 0.375rem;
+    width: 40em;
```

Comment:
Just noticed that this is using em rather than rem? Not a big deal, but seems random.

Reply:
Ahhh. So I wrote my selectors in pixels and em's and pushed the code. Late rI realized when I saw the other files that we use rem so I made another commit. I might've missed that small one. I'll add a hotfix PR and fix it

Reply:
Oh I wouldn't even bother with a hotfix.

Reply:
While you're at it can you try changing the "&#39;" in the text? I caught it earlier today but didn't think it was major enough to send another PR for

Reply:
What's wrong with that? That seems like a proper HTML encoded apostrophe?

Reply:
Nothing really, we don't really use it anywhere else so it seemed weird to have it there. It's not a big deal